browser.lisp

```diff
@@ -152,7 +152,39 @@
   "Move backwards a page."
   (webkit2:webkit-web-view-go-back (webview browser)))
 
+(defparameter *fallback-timeout* .2)
+
+;; 20150212 - MDC - Fixes issue with unavailable default scheme when opening url
```

Comment:
Granted there is probably a miniscule possibility of it occurring, but what if the project becomes decoupled with git in the future for a superior VCS system?  I'm sure there were past users of svn/bazaar that migrated to git and regretted not having explanatory comments in the codebase.

Granted, the date stamp/author initials are probably not necessary, but some code comments go a long way (as well as for someone new hopping into the project - attempting to 'git blame' every file to see why each line is there is very cumbersome compared to comments).

Thoughts?